Extremem/README.md

```diff
@@ -28,8 +28,11 @@ Command-line arguments configure Extremem to represent different combinations of
 
 It is important to complete all initialization of all global data structures before beginning to execute the experimental workload threads.  If
 initialization is still running when the workload begins to execute, the
-simulation will abort.  Increase this value if the default delays are not
-sufficient.
+simulation will abort.  This value is added to the simulation startup delay
```

Comment:
Line breaks are unneeded

Reply:
I realize the file formatting is a bit inconsistent with regards to line breaks.  But I believe the markup renderer doesn't care.